benchmarks/libseff/parsing_dollars/main.c

```diff
@@ -0,0 +1,114 @@
+#include "seff.h"
+#include <stdio.h>
+
+DEFINE_EFFECT(read, 0, char, {});
+DEFINE_EFFECT(emit, 1, void, { int64_t value; });
+DEFINE_EFFECT(stop, 2, void, {});
+
+// Improves readability, but is not needed
+#define HANDLE(func, arg, handler) handler(seff_coroutine_new(func, arg))
+
+#define NEWLINE 10
+#define DOLLAR 36
+inline bool is_newline(char c) { return c == NEWLINE; }
+inline bool is_dollar(char c) { return c == DOLLAR; }
```

Comment:
Be explicit about type conversions/casts.
```suggestion
inline bool is_newline(char c) { return (int)c == NEWLINE; }
inline bool is_dollar(char c) { return (int)c == DOLLAR; }
```